src/category_theory/FinType.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2020 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta, Adam Topaz
+-/
+
+import data.fintype.basic
+import data.fin
+import category_theory.concrete_category.bundled
+import category_theory.concrete_category
+import category_theory.full_subcategory
+import category_theory.skeletal
+
+/-!
+# The category of finite types.
+
+We define the category of finite types, denoted `Fintype` as
```

Comment:
Don't forget to rename the file to `Fintype.lean` (and change the PR title / comment while you're at it).

Reply:
Oops! Thanks :)

Reply:
Done!